calin/nz/_posts/2022-06-26-overview.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: The Start of a Journey
+author: Călin Cașcaval
+---
+
+
+While saying goodbye to a number of friends on the latest hike I organized for our
+BARD hiking group, someone suggested that I post stories and photos of our new
+planned adventure in New Zealand. So I decided to start journaling -- something that
+I have been considering for a while, but I did not have a good incentive. While we
+have been spending time outdoors, hiking, biking, and skiing, and I have posted a
+number of my pictures either on
+[500.px](https://500px.com/CalinCascaval){:target="_blank"} or shared privately,
+writing text about those has been not something I considered. After all, we want
+these hikes and skiing trips to be uneventful! So here we are, at the beginning of a
+new journey, one that feels exciting and gives me a good reason to challenge myself
+to write.
+
+Of course, everyone is asking why are we moving? The answer is ... complicated: it's
+been a while that we have been considering moving out of the US. We had an amazing
+run of 29 years here, and we are grateful for what this country has offered us:
+
+- **free education**, and actually even better -- we were paid to go to school,   learn, and earn our degrees.
+- **family life**: the opportunity to build a family; we have two loving kids who continue to teach and inspire us.
+- **job opportunities** in industries that have been growing exponentially and allowed us to live the "American dream": work hard and have a comfortable life.
+- **acceptance**: we never felt excluded or oppressed, even though we have  _funny accents_. It used to be that folks used that opportunity to ask   where are we from. It is also true that we _fit in_ -- smack in the middle of the white, middle class, educated class.
+- **natural beauty** to pursue our passion for the outdoors.
```

Comment:
*accessible* natural beauty at that, and lots of advocates for preserving it, the types of people that build the OSPs and volunteer to maintain the trails. I guess we did not interact with that community all too much, but we definitely benefited from their labors many a weekend

Reply:
Yeah, this is one that we should have done more.